### PR TITLE
Update CLDR data to v27.0.3

### PR DESCRIPTION
**Pull request by Turbo87**
see commits

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by Turbo87** [comment]
looks like the tests are not running properly. I'll have another look...

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by Turbo87** [comment]
It seems that the current CLDR data has currently incompatible issues. e.g.:

- `ca-ES` is replaced by `ca-ES-VALENCIA`
- `en-US` is replaced by `en-US-POSIX`

Unfortunately the old locales don't exist anymore and result in `No locale data` errors. Querying for the new locales doesn't work either though, since `en-US-POSIX` will result in parent locale `en-US`, which does not exist anymore.

I am not sure how to solve those issues. If you have any ideas on that please let me know.

**Comment by caridy** [comment]
I will take a look into this tomorrow.

**Comment by Turbo87** [comment]
@caridy FYI I rebased this onto #7 to get better test output for now

**Comment by Turbo87** [comment]
@caridy I think I solved it. It looks like https://github.com/unicode-cldr/cldr-core/blob/master/defaultContent.json is the solution to this problem (see third commit)

File: README.md [hunk 1]
```diff
@@ -81,14 +81,15 @@ Boolean for whether or not `fields` should be extracted for the specified `local
 
 ## Updating the CLDR Data
```
Comment (caridy) [inline README.md]: remove this topic.

**Comment by caridy** [comment]
package.json can also be cleanup, most likely. We will need to try this new version with some of the consumer pkgs to make sure that it is producing the same, or similar data.

**Comment by Turbo87** [comment]
@caridy I removed the section from the `README` and fixed the `replace()` call

**Comment by Turbo87** [comment]
@caridy any progress on this?

**Comment by caridy** [comment]
@Turbo87 I need to finish with the Intl stuff first, I have found few things that have changed in 27, I'm getting that ready before picking this up. Btw, I will be on the road next week, slowing things a little bit ;)

**Comment by Turbo87** [comment]
No problem. Just wanted to make sure you're not waiting for me to do
something. ;)

**Pull request closed without merging**